src/main/xerus/monstercat/Settings.kt

```diff
@@ -42,6 +43,26 @@ object Settings : SettingsNode("xerus/monsterutilities") {
 	
 	val FILENAMEPATTERN = create("updatePattern", "MonsterUtilities-%version%.jar")
 	
+	enum class ConnectionSpeed(val maxConnections: Int, val display: String) : Named {
+		DIALUP(5, "Dial-up (100 kb/s)"),
+		ADSL(30, "ADSL / 3G (10 Mb/s)"),
+		CABLE(150, "Cable / 4G (100 Mb/s)"),
+		FIBER(300, "Fiber (300+ Mb/s)");
+
+		override val displayName: String
+			get() = display
+
+		override fun toString(): String = display
+		
+		companion object {
+			fun findFromValue(maxConnections: Int) =
+					ConnectionSpeed.values().find { it.maxConnections == maxConnections } ?: ADSL
+			
+			fun findFromString(string: String) = ConnectionSpeed.values().find { it.toString() == string } ?: ADSL
+		}
+	}
+	val CONNECTIONSPEED = create("connectionSpeed", ConnectionSpeed.ADSL)
```

Comment:
all config values should be together, please move the enum down.

Reply:
eaeaedfcf9a64f37ac1b0ddbdd2b60139641d54b

Reply:
Pleasr also logically group the config value. Somewhere with Downloader values seems reasonable.

Reply:
Created groups here ad5b0ce049377e17fcf036eac8e9cf4a8061981c
Is that good for you ?

Reply:
:+1: